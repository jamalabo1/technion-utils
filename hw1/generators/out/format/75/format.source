A W 851 1
D Y 560 2
B W 589 3
A Y 727 4
B Y 24 5
A Z 117 6
B X 960 7
A X 934 8
B W 853 9
C Z 842 10
A Z 902 11
C W 298 12
D X 0 13
A Z 748 14
C X 134 15
A Z 937 16
A X 474 17
B Y 340 18
C Y 5 19
A W 962 20
C W 919 21
D Z 463 22
C W 350 23
D W 414 24
C W 719 25
C X 668 26
D Y 233 27
C Z 113 28
B X 615 29
B Y 629 30
D X 801 31
A X 881 32
B Z 81 33
D Y 164 34
D W 299 35
C Z 125 36
D Z 830 37
A Y 216 38
D Y 458 39
C Y 499 40
B Y 202 41
B Z 294 42
A X 611 43
A Z 942 44
D W 691 45
C Z 82 46
D W 235 47
B Z 700 48
C W 861 49
C X 725 50
D X 689 51
A Z 822 52
A W 987 53
C W 147 54
A Z 57 55
D X 248 56
A W 567 57
C Y 576 58
C Y 761 59
A X 683 60
A Z 41 61
A Y 876 62
A W 325 63
C X 869 64
C Z 656 65
A X 794 66
C Y 451 67
C X 518 68
D Y 545 69
B X 962 70
A Y 87 71
C W 930 72
C W 405 73
A Y 582 74
A Z 556 75
C Y 83 76
A Y 214 77
B Y 619 78
A Y 800 79
A Z 248 80
D Y 988 81
C Z 625 82
C Y 201 83
C Z 814 84
A W 985 85
C Y 354 86
D Y 342 87
B Z 187 88
C Y 852 89
A Y 627 90
B W 923 91
B X 220 92
D Z 106 93
B X 449 94
D Y 929 95
C X 23 96
D X 273 97
A X 877 98
C X 134 99
C X 773 100
B W 461 101
C Z 21 102
B Z 270 103
C Z 978 104
A X 425 105
A X 355 106
A Z 314 107
C X 707 108
A W 195 109
C X 57 110
B X 951 111
C Y 938 112
A W 886 113
A X 406 114
C X 675 115
C Y 1000 116
B Y 789 117
B W 12 118
D Y 84 119
B W 802 120
B W 258 121
A Y 733 122
C Z 31 123
A Z 618 124
C Y 126 125
D W 863 126
B Y 839 127
C Z 235 128
A X 760 129
B W 737 130
C Z 230 131
B Y 933 132
C Y 502 133
B Z 781 134
A X 271 135
B Z 644 136
B Y 615 137
C Z 996 138
A Z 205 139
A Z 487 140
A Z 530 141
C X 674 142
D X 465 143
C X 828 144
C X 236 145
B X 553 146
D Y 875 147
A X 25 148
D W 695 149
D X 436 150
D Z 970 151
C X 639 152
C Y 989 153
C Y 540 154
C Z 648 155
B X 855 156
A W 241 157
A X 1000 158
C W 229 159
C X 941 160
D X 246 161
A X 49 162
C Z 982 163
C Z 222 164
A Y 466 165
D Z 505 166
B Y 477 167
B Z 879 168
A Y 157 169
A Y 414 170
D W 427 171
B Y 593 172
B Z 598 173
D W 905 174
D W 685 175
D X 35 176
B Y 510 177