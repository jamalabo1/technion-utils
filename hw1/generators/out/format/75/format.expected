Blockchain Info:
1
Sender Name: B
Receiver Name: Y
Transaction Value: 510
Transaction timestamp: 177
2
Sender Name: D
Receiver Name: X
Transaction Value: 35
Transaction timestamp: 176
3
Sender Name: D
Receiver Name: W
Transaction Value: 685
Transaction timestamp: 175
4
Sender Name: D
Receiver Name: W
Transaction Value: 905
Transaction timestamp: 174
5
Sender Name: B
Receiver Name: Z
Transaction Value: 598
Transaction timestamp: 173
6
Sender Name: B
Receiver Name: Y
Transaction Value: 593
Transaction timestamp: 172
7
Sender Name: D
Receiver Name: W
Transaction Value: 427
Transaction timestamp: 171
8
Sender Name: A
Receiver Name: Y
Transaction Value: 414
Transaction timestamp: 170
9
Sender Name: A
Receiver Name: Y
Transaction Value: 157
Transaction timestamp: 169
10
Sender Name: B
Receiver Name: Z
Transaction Value: 879
Transaction timestamp: 168
11
Sender Name: B
Receiver Name: Y
Transaction Value: 477
Transaction timestamp: 167
12
Sender Name: D
Receiver Name: Z
Transaction Value: 505
Transaction timestamp: 166
13
Sender Name: A
Receiver Name: Y
Transaction Value: 466
Transaction timestamp: 165
14
Sender Name: C
Receiver Name: Z
Transaction Value: 222
Transaction timestamp: 164
15
Sender Name: C
Receiver Name: Z
Transaction Value: 982
Transaction timestamp: 163
16
Sender Name: A
Receiver Name: X
Transaction Value: 49
Transaction timestamp: 162
17
Sender Name: D
Receiver Name: X
Transaction Value: 246
Transaction timestamp: 161
18
Sender Name: C
Receiver Name: X
Transaction Value: 941
Transaction timestamp: 160
19
Sender Name: C
Receiver Name: W
Transaction Value: 229
Transaction timestamp: 159
20
Sender Name: A
Receiver Name: X
Transaction Value: 1000
Transaction timestamp: 158
21
Sender Name: A
Receiver Name: W
Transaction Value: 241
Transaction timestamp: 157
22
Sender Name: B
Receiver Name: X
Transaction Value: 855
Transaction timestamp: 156
23
Sender Name: C
Receiver Name: Z
Transaction Value: 648
Transaction timestamp: 155
24
Sender Name: C
Receiver Name: Y
Transaction Value: 540
Transaction timestamp: 154
25
Sender Name: C
Receiver Name: Y
Transaction Value: 989
Transaction timestamp: 153
26
Sender Name: C
Receiver Name: X
Transaction Value: 639
Transaction timestamp: 152
27
Sender Name: D
Receiver Name: Z
Transaction Value: 970
Transaction timestamp: 151
28
Sender Name: D
Receiver Name: X
Transaction Value: 436
Transaction timestamp: 150
29
Sender Name: D
Receiver Name: W
Transaction Value: 695
Transaction timestamp: 149
30
Sender Name: A
Receiver Name: X
Transaction Value: 25
Transaction timestamp: 148
31
Sender Name: D
Receiver Name: Y
Transaction Value: 875
Transaction timestamp: 147
32
Sender Name: B
Receiver Name: X
Transaction Value: 553
Transaction timestamp: 146
33
Sender Name: C
Receiver Name: X
Transaction Value: 236
Transaction timestamp: 145
34
Sender Name: C
Receiver Name: X
Transaction Value: 828
Transaction timestamp: 144
35
Sender Name: D
Receiver Name: X
Transaction Value: 465
Transaction timestamp: 143
36
Sender Name: C
Receiver Name: X
Transaction Value: 674
Transaction timestamp: 142
37
Sender Name: A
Receiver Name: Z
Transaction Value: 530
Transaction timestamp: 141
38
Sender Name: A
Receiver Name: Z
Transaction Value: 487
Transaction timestamp: 140
39
Sender Name: A
Receiver Name: Z
Transaction Value: 205
Transaction timestamp: 139
40
Sender Name: C
Receiver Name: Z
Transaction Value: 996
Transaction timestamp: 138
41
Sender Name: B
Receiver Name: Y
Transaction Value: 615
Transaction timestamp: 137
42
Sender Name: B
Receiver Name: Z
Transaction Value: 644
Transaction timestamp: 136
43
Sender Name: A
Receiver Name: X
Transaction Value: 271
Transaction timestamp: 135
44
Sender Name: B
Receiver Name: Z
Transaction Value: 781
Transaction timestamp: 134
45
Sender Name: C
Receiver Name: Y
Transaction Value: 502
Transaction timestamp: 133
46
Sender Name: B
Receiver Name: Y
Transaction Value: 933
Transaction timestamp: 132
47
Sender Name: C
Receiver Name: Z
Transaction Value: 230
Transaction timestamp: 131
48
Sender Name: B
Receiver Name: W
Transaction Value: 737
Transaction timestamp: 130
49
Sender Name: A
Receiver Name: X
Transaction Value: 760
Transaction timestamp: 129
50
Sender Name: C
Receiver Name: Z
Transaction Value: 235
Transaction timestamp: 128
51
Sender Name: B
Receiver Name: Y
Transaction Value: 839
Transaction timestamp: 127
52
Sender Name: D
Receiver Name: W
Transaction Value: 863
Transaction timestamp: 126
53
Sender Name: C
Receiver Name: Y
Transaction Value: 126
Transaction timestamp: 125
54
Sender Name: A
Receiver Name: Z
Transaction Value: 618
Transaction timestamp: 124
55
Sender Name: C
Receiver Name: Z
Transaction Value: 31
Transaction timestamp: 123
56
Sender Name: A
Receiver Name: Y
Transaction Value: 733
Transaction timestamp: 122
57
Sender Name: B
Receiver Name: W
Transaction Value: 258
Transaction timestamp: 121
58
Sender Name: B
Receiver Name: W
Transaction Value: 802
Transaction timestamp: 120
59
Sender Name: D
Receiver Name: Y
Transaction Value: 84
Transaction timestamp: 119
60
Sender Name: B
Receiver Name: W
Transaction Value: 12
Transaction timestamp: 118
61
Sender Name: B
Receiver Name: Y
Transaction Value: 789
Transaction timestamp: 117
62
Sender Name: C
Receiver Name: Y
Transaction Value: 1000
Transaction timestamp: 116
63
Sender Name: C
Receiver Name: X
Transaction Value: 675
Transaction timestamp: 115
64
Sender Name: A
Receiver Name: X
Transaction Value: 406
Transaction timestamp: 114
65
Sender Name: A
Receiver Name: W
Transaction Value: 886
Transaction timestamp: 113
66
Sender Name: C
Receiver Name: Y
Transaction Value: 938
Transaction timestamp: 112
67
Sender Name: B
Receiver Name: X
Transaction Value: 951
Transaction timestamp: 111
68
Sender Name: C
Receiver Name: X
Transaction Value: 57
Transaction timestamp: 110
69
Sender Name: A
Receiver Name: W
Transaction Value: 195
Transaction timestamp: 109
70
Sender Name: C
Receiver Name: X
Transaction Value: 707
Transaction timestamp: 108
71
Sender Name: A
Receiver Name: Z
Transaction Value: 314
Transaction timestamp: 107
72
Sender Name: A
Receiver Name: X
Transaction Value: 355
Transaction timestamp: 106
73
Sender Name: A
Receiver Name: X
Transaction Value: 425
Transaction timestamp: 105
74
Sender Name: C
Receiver Name: Z
Transaction Value: 978
Transaction timestamp: 104
75
Sender Name: B
Receiver Name: Z
Transaction Value: 270
Transaction timestamp: 103
76
Sender Name: C
Receiver Name: Z
Transaction Value: 21
Transaction timestamp: 102
77
Sender Name: B
Receiver Name: W
Transaction Value: 461
Transaction timestamp: 101
78
Sender Name: C
Receiver Name: X
Transaction Value: 773
Transaction timestamp: 100
79
Sender Name: C
Receiver Name: X
Transaction Value: 134
Transaction timestamp: 99
80
Sender Name: A
Receiver Name: X
Transaction Value: 877
Transaction timestamp: 98
81
Sender Name: D
Receiver Name: X
Transaction Value: 273
Transaction timestamp: 97
82
Sender Name: C
Receiver Name: X
Transaction Value: 23
Transaction timestamp: 96
83
Sender Name: D
Receiver Name: Y
Transaction Value: 929
Transaction timestamp: 95
84
Sender Name: B
Receiver Name: X
Transaction Value: 449
Transaction timestamp: 94
85
Sender Name: D
Receiver Name: Z
Transaction Value: 106
Transaction timestamp: 93
86
Sender Name: B
Receiver Name: X
Transaction Value: 220
Transaction timestamp: 92
87
Sender Name: B
Receiver Name: W
Transaction Value: 923
Transaction timestamp: 91
88
Sender Name: A
Receiver Name: Y
Transaction Value: 627
Transaction timestamp: 90
89
Sender Name: C
Receiver Name: Y
Transaction Value: 852
Transaction timestamp: 89
90
Sender Name: B
Receiver Name: Z
Transaction Value: 187
Transaction timestamp: 88
91
Sender Name: D
Receiver Name: Y
Transaction Value: 342
Transaction timestamp: 87
92
Sender Name: C
Receiver Name: Y
Transaction Value: 354
Transaction timestamp: 86
93
Sender Name: A
Receiver Name: W
Transaction Value: 985
Transaction timestamp: 85
94
Sender Name: C
Receiver Name: Z
Transaction Value: 814
Transaction timestamp: 84
95
Sender Name: C
Receiver Name: Y
Transaction Value: 201
Transaction timestamp: 83
96
Sender Name: C
Receiver Name: Z
Transaction Value: 625
Transaction timestamp: 82
97
Sender Name: D
Receiver Name: Y
Transaction Value: 988
Transaction timestamp: 81
98
Sender Name: A
Receiver Name: Z
Transaction Value: 248
Transaction timestamp: 80
99
Sender Name: A
Receiver Name: Y
Transaction Value: 800
Transaction timestamp: 79
100
Sender Name: B
Receiver Name: Y
Transaction Value: 619
Transaction timestamp: 78
101
Sender Name: A
Receiver Name: Y
Transaction Value: 214
Transaction timestamp: 77
102
Sender Name: C
Receiver Name: Y
Transaction Value: 83
Transaction timestamp: 76
103
Sender Name: A
Receiver Name: Z
Transaction Value: 556
Transaction timestamp: 75
104
Sender Name: A
Receiver Name: Y
Transaction Value: 582
Transaction timestamp: 74
105
Sender Name: C
Receiver Name: W
Transaction Value: 405
Transaction timestamp: 73
106
Sender Name: C
Receiver Name: W
Transaction Value: 930
Transaction timestamp: 72
107
Sender Name: A
Receiver Name: Y
Transaction Value: 87
Transaction timestamp: 71
108
Sender Name: B
Receiver Name: X
Transaction Value: 962
Transaction timestamp: 70
109
Sender Name: D
Receiver Name: Y
Transaction Value: 545
Transaction timestamp: 69
110
Sender Name: C
Receiver Name: X
Transaction Value: 518
Transaction timestamp: 68
111
Sender Name: C
Receiver Name: Y
Transaction Value: 451
Transaction timestamp: 67
112
Sender Name: A
Receiver Name: X
Transaction Value: 794
Transaction timestamp: 66
113
Sender Name: C
Receiver Name: Z
Transaction Value: 656
Transaction timestamp: 65
114
Sender Name: C
Receiver Name: X
Transaction Value: 869
Transaction timestamp: 64
115
Sender Name: A
Receiver Name: W
Transaction Value: 325
Transaction timestamp: 63
116
Sender Name: A
Receiver Name: Y
Transaction Value: 876
Transaction timestamp: 62
117
Sender Name: A
Receiver Name: Z
Transaction Value: 41
Transaction timestamp: 61
118
Sender Name: A
Receiver Name: X
Transaction Value: 683
Transaction timestamp: 60
119
Sender Name: C
Receiver Name: Y
Transaction Value: 761
Transaction timestamp: 59
120
Sender Name: C
Receiver Name: Y
Transaction Value: 576
Transaction timestamp: 58
121
Sender Name: A
Receiver Name: W
Transaction Value: 567
Transaction timestamp: 57
122
Sender Name: D
Receiver Name: X
Transaction Value: 248
Transaction timestamp: 56
123
Sender Name: A
Receiver Name: Z
Transaction Value: 57
Transaction timestamp: 55
124
Sender Name: C
Receiver Name: W
Transaction Value: 147
Transaction timestamp: 54
125
Sender Name: A
Receiver Name: W
Transaction Value: 987
Transaction timestamp: 53
126
Sender Name: A
Receiver Name: Z
Transaction Value: 822
Transaction timestamp: 52
127
Sender Name: D
Receiver Name: X
Transaction Value: 689
Transaction timestamp: 51
128
Sender Name: C
Receiver Name: X
Transaction Value: 725
Transaction timestamp: 50
129
Sender Name: C
Receiver Name: W
Transaction Value: 861
Transaction timestamp: 49
130
Sender Name: B
Receiver Name: Z
Transaction Value: 700
Transaction timestamp: 48
131
Sender Name: D
Receiver Name: W
Transaction Value: 235
Transaction timestamp: 47
132
Sender Name: C
Receiver Name: Z
Transaction Value: 82
Transaction timestamp: 46
133
Sender Name: D
Receiver Name: W
Transaction Value: 691
Transaction timestamp: 45
134
Sender Name: A
Receiver Name: Z
Transaction Value: 942
Transaction timestamp: 44
135
Sender Name: A
Receiver Name: X
Transaction Value: 611
Transaction timestamp: 43
136
Sender Name: B
Receiver Name: Z
Transaction Value: 294
Transaction timestamp: 42
137
Sender Name: B
Receiver Name: Y
Transaction Value: 202
Transaction timestamp: 41
138
Sender Name: C
Receiver Name: Y
Transaction Value: 499
Transaction timestamp: 40
139
Sender Name: D
Receiver Name: Y
Transaction Value: 458
Transaction timestamp: 39
140
Sender Name: A
Receiver Name: Y
Transaction Value: 216
Transaction timestamp: 38
141
Sender Name: D
Receiver Name: Z
Transaction Value: 830
Transaction timestamp: 37
142
Sender Name: C
Receiver Name: Z
Transaction Value: 125
Transaction timestamp: 36
143
Sender Name: D
Receiver Name: W
Transaction Value: 299
Transaction timestamp: 35
144
Sender Name: D
Receiver Name: Y
Transaction Value: 164
Transaction timestamp: 34
145
Sender Name: B
Receiver Name: Z
Transaction Value: 81
Transaction timestamp: 33
146
Sender Name: A
Receiver Name: X
Transaction Value: 881
Transaction timestamp: 32
147
Sender Name: D
Receiver Name: X
Transaction Value: 801
Transaction timestamp: 31
148
Sender Name: B
Receiver Name: Y
Transaction Value: 629
Transaction timestamp: 30
149
Sender Name: B
Receiver Name: X
Transaction Value: 615
Transaction timestamp: 29
150
Sender Name: C
Receiver Name: Z
Transaction Value: 113
Transaction timestamp: 28
151
Sender Name: D
Receiver Name: Y
Transaction Value: 233
Transaction timestamp: 27
152
Sender Name: C
Receiver Name: X
Transaction Value: 668
Transaction timestamp: 26
153
Sender Name: C
Receiver Name: W
Transaction Value: 719
Transaction timestamp: 25
154
Sender Name: D
Receiver Name: W
Transaction Value: 414
Transaction timestamp: 24
155
Sender Name: C
Receiver Name: W
Transaction Value: 350
Transaction timestamp: 23
156
Sender Name: D
Receiver Name: Z
Transaction Value: 463
Transaction timestamp: 22
157
Sender Name: C
Receiver Name: W
Transaction Value: 919
Transaction timestamp: 21
158
Sender Name: A
Receiver Name: W
Transaction Value: 962
Transaction timestamp: 20
159
Sender Name: C
Receiver Name: Y
Transaction Value: 5
Transaction timestamp: 19
160
Sender Name: B
Receiver Name: Y
Transaction Value: 340
Transaction timestamp: 18
161
Sender Name: A
Receiver Name: X
Transaction Value: 474
Transaction timestamp: 17
162
Sender Name: A
Receiver Name: Z
Transaction Value: 937
Transaction timestamp: 16
163
Sender Name: C
Receiver Name: X
Transaction Value: 134
Transaction timestamp: 15
164
Sender Name: A
Receiver Name: Z
Transaction Value: 748
Transaction timestamp: 14
165
Sender Name: D
Receiver Name: X
Transaction Value: 0
Transaction timestamp: 13
166
Sender Name: C
Receiver Name: W
Transaction Value: 298
Transaction timestamp: 12
167
Sender Name: A
Receiver Name: Z
Transaction Value: 902
Transaction timestamp: 11
168
Sender Name: C
Receiver Name: Z
Transaction Value: 842
Transaction timestamp: 10
169
Sender Name: B
Receiver Name: W
Transaction Value: 853
Transaction timestamp: 9
170
Sender Name: A
Receiver Name: X
Transaction Value: 934
Transaction timestamp: 8
171
Sender Name: B
Receiver Name: X
Transaction Value: 960
Transaction timestamp: 7
172
Sender Name: A
Receiver Name: Z
Transaction Value: 117
Transaction timestamp: 6
173
Sender Name: B
Receiver Name: Y
Transaction Value: 24
Transaction timestamp: 5
174
Sender Name: A
Receiver Name: Y
Transaction Value: 727
Transaction timestamp: 4
175
Sender Name: B
Receiver Name: W
Transaction Value: 589
Transaction timestamp: 3
176
Sender Name: D
Receiver Name: Y
Transaction Value: 560
Transaction timestamp: 2
177
Sender Name: A
Receiver Name: W
Transaction Value: 851
Transaction timestamp: 1